developer-docs-site/docs/tutorials/first-move-module.md

```diff
@@ -38,6 +38,9 @@ For this tutorial, will be focusing on `hello_blockchain/src` and re-using the `
 
 You can find the rust project [here](https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site/static/examples/rust)
```

Comment:
rust → Rust

Reply:
Put a period at the end of either "here" as `[here.]` or after the closing bracket of the link.